olive/passes/onnx/optimum_conversion.py

```diff
@@ -21,10 +21,29 @@ class OptimumConversion(Pass):
 
     @staticmethod
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
+        device = "cuda" if accelerator_spec.accelerator_type == "gpu" else "cpu"
         config = {
             "target_opset": PassConfigParam(
                 type_=int, default_value=14, description="The version of the default (ai.onnx) opset to target."
-            )
+            ),
+            "fp16": PassConfigParam(
+                type_=bool,
+                default_value=False,
+                description="Whether to use fp16 precision to load torch model and then convert it to onnx.",
+            ),
+            "device": PassConfigParam(
```

Comment:
Should we set the default device to be `None` (or cpu) like with the conversion pass and make the user specify their own device if needed? Just like in conversion, we might want have gpu target but do conversion on CPU due to memory limits, etc.

Reply:
Conversion on cpu still seems to be recommended. GPU is only required for fp16